google-cloud-bigquery/lib/google/cloud/bigquery/schema/field.rb

```diff
@@ -163,6 +163,62 @@ def mode= new_mode
             @gapi.update! mode: verify_mode(new_mode)
           end
 
+          ##
+          # The policy tag list for the field. Policy tag identifiers are of the form
+          # `projects/*/locations/*/taxonomies/*/policyTags/*`.
+          #
+          # @see https://cloud.google.com/bigquery/docs/column-level-security-intro
+          #   Introduction to BigQuery column-level security
+          #
+          # @return [Array<String>, nil] The policy tag list for the field, or `nil`.
+          #
+          # @example
+          #   require "google/cloud/bigquery"
+          #
+          #   bigquery = Google::Cloud::Bigquery.new
+          #   dataset = bigquery.dataset "my_dataset"
+          #   table = dataset.table "my_table"
+          #
+          #   table.schema.field("age").policy_tags
+          #
+          def policy_tags
+            names = @gapi.policy_tags&.names
+            names.to_a if names && !names.empty?
+          end
+
+          ##
+          # Updates the policy tag list for the field.
+          #
+          # @see https://cloud.google.com/bigquery/docs/column-level-security-intro
+          #   Introduction to BigQuery column-level security
+          #
+          # @param [Array<String>, String, nil] new_policy_tags The policy tag list or
+          #   single policy tag for the field, or `nil` to remove the existing policy tags.
+          #   Policy tag identifiers are of the form
+          #   `projects/*/locations/*/taxonomies/*/policyTags/*`.
```

Comment:
Do we need to include the "At most 1 policy tag is currently allowed" caveat here as well?

Reply:
Yes, thanks for catching that!